src/hr/setup/organization/organization.controller.ts

```diff
@@ -0,0 +1,113 @@
+import {
+  Body,
+  Controller,
+  Get,
+  HttpStatus,
+  Param,
+  Patch,
+  Post,
+  UseGuards,
+} from '@nestjs/common';
+
+import { Users } from '@prisma/client';
+import { AuthGuard } from '@nestjs/passport';
+import { AuthUserInfo } from 'src/decorator/auth.user.info.decorator';
+import { OrganizationService } from './organization.service';
+import { CreateOrganizationDto } from './dto/create.organization.dto';
+import { UpdateOrganizationDto } from './dto/update.organization.dto';
+
+@Controller('organization')
+export class OrganizationController {
+  constructor(private readonly organizationService: OrganizationService) {}
+
+  @UseGuards(AuthGuard('jwt'))
+  @Get()
+  async getAll() {
+    try {
+      const results = await this.organizationService.getAll();
+      return {
+        message: 'Show data successfully',
```

Comment:
`message` property pass korar dorkar nai response e data fetching er shomoy, create, update and delete e pass korte paren